README.md

```diff
@@ -2,3 +2,19 @@
 
 [![Build Status](https://travis-ci.org/hmcts/bulk-scan-orchestrator.svg?branch=master)](https://travis-ci.org/hmcts/bulk-scan-orchestrator)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e9272daf4b714e4f95280916e763b6b2)](https://www.codacy.com/app/HMCTS/bulk-scan-orchestrator)
+
+## Running end to end using docker
+- Run up the docker environment from bulk-scan-shared-infrastructure
+- You will have to setup these environment variables on either your
+run configuration or bash shell
+```$bash
+IDAM_USERS_SSCS_USERNAME = bulkscanorchestrator+systemupdate@gmail.com
+IDAM_USERS_SSCS_PASSWORD = Password12
+CORE_CASE_DATA_API_URL = http://localhost:4452
```

Comment:
isn't this already set as the default?

Reply:
only if you have the application-default.yaml set.

Reply:
oh why did you remove the config?
i.e. we had it defaulted before to config that would work for running in your IDE against docker.

Reply:
Thought that is what you asked for ... hmmm.

Reply:
nah I meant don't override the property with an env var and a default, use the env var that spring accepts in the environment and the default locally,
something like:
```
core_case_data:
  api:
    url: http://localhost:4452
```